leakcanary-object-watcher-android/src/main/java/leakcanary/AppWatcher.kt

```diff
@@ -106,6 +106,9 @@ object AppWatcher {
    * or LeakCanary to run outside of the main process then you can call this method to install
    * [AppWatcher].
    */
-  fun manualInstall(application: Application) = InternalAppWatcher.install(application)
+  fun manualInstall(application: Application) {
+    config = Config()
```

Comment:
The existing config may already have been customized, so we shouldn't replace it with a brand new default config. try this instead:

```
config = config.copy(enabled = true)
```

Reply:
Thanks for the advice.

I've changed it to:
`config = config.copy(enabled = true)`

But I have doubt whether it needs to set the enabled value by application debuggable value.

Now we set the enabled value to true and suppose that people who call it before should know that `manualInstall` will works immediately. This sounds make sense too.